### PR TITLE
[On Hold] Demonstrate usage of SWC loader

### PR DESCRIPTION
**Pull request by tomdracz**
For more info, see https://github.com/shakacode/shakapacker/pull/29

Demonstrates switching existing Shakapacker/Webpacker app to use SWC loader instead of babel loader.

All that is needed:
- Use correct version of Shakapacker (currently pointed at my fork)
- Install `@swc/core` and `swc-loader` to package.json dependencies
- Add `webpack_loader: swc` to `webpacker.yml` configuration
- Add custom `config/swc.config.js` to enable fast refresh (not enabled by default in Shakapacker)

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails_demo_ssr_hmr/30)

### CODE REVIEW

File: config/webpacker.yml [hunk 1]
```diff
@@ -15,6 +15,8 @@ default: &default
   # Reload manifest.json on all requests so we reload latest compiled packs
   cache_manifest: false
 
+  webpack_loader: swc
```
Comment (justin808) [inline config/webpacker.yml:18]: 👍

**Review by justin808 — COMMENTED** [review]
We'll merge this once we ship a version!

**Comment by justin808** [comment]
We should update this PR to check that SWC still works with Shakapacker v7.

@ahangarha @tomdracz

**Comment by ahangarha** [comment]
@justin808 
Please check #56

**Pull request closed without merging**